database/datamodel.graphql

```diff
@@ -1,86 +1,86 @@
 enum Role {
-    MEMBER
```

Comment:
@phuongthuan next time, you can create a separated pull request just for code format if there is a lot of spacing changes